### ISSUE TITLE
NameError: name 'cyclib' is not defined

### ISSUE THREAD

**Issue by bam241** [cyclus/cyclus#1388]
After reinstalling `cyclus` I got this message when running the unit test:

```
Exception ignored in: 'eventhooks.eventloophook'
Traceback (most recent call last):
  File "/Users/mouginot/.local/lib/python3.5/site-packages/cyclus/__init__.py", line 5, in <module>
    from cyclus.lib import py_init_hooks
ImportError: cannot import name 'py_init_hooks'
Exception ignored in: 'eventhooks.eventloophook'
Traceback (most recent call last):
  File "/Users/mouginot/.local/lib/python3.5/site-packages/cyclus/__init__.py", line 5, in <module>
    from cyclus.lib import py_init_hooks
ImportError: cannot import name 'py_init_hooks'
Exception ignored in: 'pymodule.py_del_agent'
NameError: name 'cyclib' is not defined
```

I don't know if this is related to #1366 or if this is a separated issue, or if this is related to my configuration...

**Comment by bam241** [comment]
note that the unit_test are not failing.

**Comment by katyhuff** [comment]
we think (thanks to @scopatz ) this is a deep python import error related to not having pandas.

**Comment by scopatz** [comment]
And to @bam241 ;)

**Comment by katyhuff** [comment]
This is closed via #1391 and #1394 .
